atomic_reactor/utils/resource_manager.py

```diff
@@ -0,0 +1,187 @@
+"""
+Copyright (c) 2022 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+import os
+import logging
+import json
+import paramiko
+from datetime import datetime
+from shlex import quote
+
+ROOT_PATH = 'resource_manager/'
+logger = logging.getLogger(__name__)
+
+
+class Host:
+    '''
+    Remote host abstract class
+    '''
+    def __init__(self, username: str, hostname: str, arch: str):
+        self.username = username
+        self.hostname = hostname
+        self.arch = arch
+        self.slots = []
+        self.free_slots = 0
+
+    def _read_info_data(self):
+        cmd = f"cat {ROOT_PATH}info.json"
```

Comment:
These commands retrieves content of json files. Why do not we return from this method dictionary via `return json.loads(data)`? The rest of the code would be easier to read and you would spare plenty of lines.